cmd/server/main.go

```diff
@@ -19,10 +22,11 @@ func main() {
 	}
 	port := flag.Int("port", 8080, "Port number")
 	dataSourceName := flag.String("data-source-name", "", "DB URL")
+	retention := flag.Duration("retention", 0, "Duration of metrics retention")
```

Comment:
I'm not sure how these are set in our deployment pipeline. Better to make the default what we'd want it set to now.

Reply:
Since setting it to 0 will just keep retention disabled, it should be fine for now